### ISSUE TITLE
Incorrect url comment notes in generated scaffold controller in a namespace.

### ISSUE THREAD

**Issue by qichunren** [rails/rails#44662]
### Steps to reproduce

```ruby
rails new test_blog
cd test_blog
./bin/rails generate model Post title context:text
./bin/rails g scaffold_controller Admin::Post title context:text --model-name=Post
```

### Expected behavior

Every generated url comment note should contain namespace prefix like: `# GET /admin/posts or /admin/posts.json`

### Actual behavior

Every generated url comment note missing namespace prefix like: `# GET /posts or /posts.json`

### System configuration
**Rails version**: v7.0.2.2

**Ruby version**: v3.1.0

**Comment by pixeltrix** [comment]
@qichunren according to the USAGE file, you're meant to use `module/class` and not `module::class`, e.g.

```
bin/rails g scaffold_controller Admin/Post
```

This generates the correct comments when I test locally.

**Comment by qichunren** [comment]
@pixeltrix I test locally with your way, `./bin/rails g scaffold_controller Admin/Post title context:text --model-name=Post`,  it still generates incorrect comments as the issue reported. Did I miss something?

**Comment by pixeltrix** [comment]
> Did I miss something?

No, I missed adding the `--model-name` option 😞 

I made a PR to fix this: #44670
